lambdas/data-migration2/src/executions.ts

```diff
@@ -79,6 +82,8 @@ export const migrateExecutions = async (
     failed: 0,
     skipped: 0,
   };
+  const errorFile = [];
+  let errorMessage;
```

Comment:
Why are we initializing the errorMessage variable here? The variable seems to be only used inside the else block below, so I would prefer to use `const` inside that block